src/model.rs

```diff
@@ -896,48 +902,63 @@ impl Collections {
             }
             fn find_stop_area_for<'a, F>(
```

Comment:
I changed the name of `find_biggest_stop_area` to `find_biggest_stop_area**s**` because it makes sense (see 40b56c42ced51b5cf1e29005353f567dde60113d). However, I don't think I'll do the other suggested change (`find_stop_area_for` into `find_biggest_stop_area_for`) because it would be incorrect. If we want to go that path, it would be more correct with `find_most_frequent_biggest_alphabetical_stop_area_for` but I'm not sure we want that much.

Reply:
OK, I agree for no such long name :-)
I can suggest find_`best`_stop_area_for or find_`most_important`_stop_area_for if you think one is a good candidate, otherwise you can resolve.

Reply:
I like `find_best_stop_area_for`. Done in f98067cc1227e7f181baa9eeb4e28b34e16a6b14.